arch/xtensa/include/syscall.h

```diff
@@ -29,10 +29,34 @@
  * Included Files
  ****************************************************************************/
 
+#include <nuttx/config.h>
+#include <debug.h>
+#include <arch/irq.h>
+#ifndef __ASSEMBLY__
+#  include <stdint.h>
+#endif
+
+#include "xtensa_swi.h"
```

Comment:
@zhuyanlinzyl I am trying to build but this file doesn't exisit.

Reply:
It's my mistake, I update the patch again.

Your can build now